corehq/ex-submodules/casexml/apps/phone/restore_caching.py

```diff
@@ -26,6 +28,16 @@ def invalidate(self):
         get_redis_default_cache().delete(self.cache_key)
 
 
+@quickcache(['domain', 'user_id'])
```

Comment:
Should this be quickcached for longer?

Reply:
The default is 5 hours. What do you think would be a reasonable timeout? 24 hours?

Reply:
I think the default is 5 minutes? not 5 hours?

Reply:
Yeah silly me you're right. I'll make it the same timeout as restore cache